api/src/main/java/org/openmrs/module/eptsreports/reporting/library/datasets/resumo/ResumoMensalDataSetDefinition.java

```diff
@@ -363,6 +363,31 @@ public DataSetDefinition constructResumoMensalDataset() {
         getPatientsWhoWereActiveByEndOfPreviousMonth(),
         resumoMensalAandBdisaggregations.getAdolescentesColumns());
 
+    // B13 Indicators
+    dsd.addColumn(
+        "B13TP",
+        "Total patients - Total Geral",
+        this.getActivePatientsInARTByEndOfCurrentMonth(),
```

Comment:
We don't need to use `this`. I bet this is an eclipse auto thing.

Reply:
ok, but  why we shouldn't  use  `this`?

Reply:
`This` is implicit, even if you don't add it, the method call will default to the class scope. Apart from that, all other code in the project omits `this` when not strictly necessary.